endless-sky.appdata.xml

```diff
@@ -42,6 +42,542 @@
     </screenshot>
   </screenshots>
   <releases>
+    <release version="0.9.11" date="2020-02-24">
```

Comment:
The odd releases should include `type="development"` and perhaps also `urgency="low"` per the spec.